main/introduction.bs

```diff
@@ -18,6 +18,12 @@ with mechanisms for controlling who can access what.
 A <dfn export>Solid app</dfn> is an application
 that reads or writes data from one or more [=data pods=].
 
+A <dfn export>read operation</dfn> entails that information can be revealed about the content of a resource, and categorically excludes operations to "write", as well as unintended exposure or side-effects within that. [[Source](https://github.com/solid/specification/issues/149#issue-568433265)]
```

Comment:
I'm not sure I understand what line 21 is saying.. Any chance to re-phrase it simpler?

Reply:
What do you think of this revision: https://github.com/solid/specification/pull/157/commits/8c28e8edd6bb6e19f21200b870736cd344a095be

Reply:
@csarven clear, thank you!